core/services/wifi/main.py

```diff
@@ -60,13 +64,7 @@ async def scan() -> Any:
             logger.info(line)
         return available_networks
     except BusyError as error:
-        logger.error("Failed to scan networks.")
-        logger.exception(error)
-        raise HTTPException(status_code=status.HTTP_425_TOO_EARLY, detail=str(error)) from error
-    except FetchError as error:
-        logger.error("Failed to scan networks.")
-        logger.exception(error)
-        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=str(error)) from error
+        raise StackedHTTPException(status_code=status.HTTP_425_TOO_EARLY, error=error) from error
```

Comment:
We are not dealing with two different erros here ? FetchError appears as 500 on the original code.

Reply:
Ping @rafaellehmkuhl

Reply:
This change will continue to catch BusyError as usual, returning a 425, while the FetchError will be treated as before (not catched thus returning a 500 error from the GenericHandler).

Reply:
This change will continue to catch BusyError as usual, returning a 425, while the FetchError will be treated as before (not catched thus returning a 500 error from the GenericHandler).